### PR TITLE
[THURSDAY AFTERNOON] Physics of living systems

### PR DESCRIPTION
**Pull request by nlisgo**
This should not be merged until we have a rewrite in place on journal.

https://github.com/elifesciences/builder-private/pull/144

### CODE REVIEW

File: src/modules/jcms_migrate/migration_assets/subjects.json [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+  },
   {
     "id": "plant-biology",
     "name": "Plant Biology",
     "impact_statement": "eLife publishes plant biology research including development, epigenetics, genomics, signalling, metabolism and plant–microbe interactions. Decisions are made by <a href=\"/about/people\">journal editors</a> who are active researchers in plant biology.",
     "image": "subjects/elife-sciences-plant-biology-illustration.jpg",
     "alt": "Plant Biology"
   },
+  {
+    "id": "physics-of-living-systems",
```
Comment (giorgiosironi) [inline src/modules/jcms_migrate/migration_assets/subjects.json]: To generate ids, we drop the `and` in names, do we want to drop `of` too?

Comment (nlisgo) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: I was unsure on this one.

Comment (giorgiosironi) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: asked; https://elifesciences.slack.com/archives/C98MV3UG7/p1519811877000337 says to go without the `of` for consistency with the others

File: src/modules/jcms_migrate/migration_assets/subjects.json [hunk 2]
```diff
@@ -1,11 +1,25 @@
 [
+  {
+    "id": "structural-biology-molecular-biophysics",
+    "name": "Structural Biology and Molecular Biophysics",
+    "impact_statement": "eLife publishes research that uses techniques including X-ray crystallography, cryo-EM and single-molecule methods. Decisions are made by <a href=\"/about/people\">journal editors</a> who are active researchers in the fields of biophysics and structural biology.",
+    "image": "subjects/elife-sciences-biophysics-and-structural-biology-illustration.jpg",
```
Comment (giorgiosironi) [inline src/modules/jcms_migrate/migration_assets/subjects.json]: maybe this image should be duplicated to use the new name too?

File: src/modules/jcms_migrate/jcms_migrate.install [hunk 3]
```diff
@@ -915,3 +915,13 @@ function jcms_migrate_update_8116() {
     $migrate_id_map->delete([$msa]);
   }
 }
+
+/**
+ * Import the MSA physics-living-systems.
+ */
+function jcms_migrate_update_8117() {
```
Comment (nlisgo) [inline src/modules/jcms_migrate/jcms_migrate.install]: This will be incremented after https://github.com/elifesciences/journal-cms/pull/280

**Pull request merged**